### ISSUE TITLE
Add App Route: Github PRs

### ISSUE THREAD

**Issue by jbenet** [jbenet/depviz#5]
Make this route work:

```
/http/github.com/<user>/<repo>/pull/<pr>
```

---

depends on #3
depends on #11

**Comment by jbenet** [comment]
depends on #3

**Comment by wking** [comment]
Done in #11 (although it redirects to the canonical form without /pull, since the distinction between issues and pull requests is not important for depviz). Can we either add a `depends on #11` or merge #11 and close this?

**Comment by jbenet** [comment]
I think we don't gain much in losing the `issue/` and `pull/` part of the canonical id/link-- given the links are quite long anyway. maybe can discuss it in another issue if it causes problems.
